resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
still don't really see the merit in maintain two separate models. a page view is just a type of event that happens when the client decides a page has been seen.